packages/core/src/drivers/db/models/diligencia.js

```diff
@@ -0,0 +1,65 @@
+const { Model, DataTypes } = require('sequelize');
+const { USUARIO_TABLE } = require('./persona');
+
+const DILIGENCIA_TABLE = 'diligencia';
+
+const DiligenciaSchema = {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER,
+  },
+  personaId: {
+    allowNull: false,
+    type: DataTypes.INTEGER,
+    field: 'persona_id',
+    references: {
+      model: USUARIO_TABLE,
+      key: 'id',
+    },
+  },
+
+  startTime: {
+    allowNull: false,
+    type: DataTypes.DATE,
+    field: 'start_time',
+  },
+
+  endTieme: {
+    allowNull: false,
+    type: DataTypes.DATE,
+    field: 'start_time',
+  },
```

Comment:
I think we will continue using the Spanish language for the fields of the tables and this field is about active hours in their jobs, not DATE. It could be a string or I don't know if there is a DataType.TIME
hora_inicio
hora_fin
o un string de 
horario_laboral: 9 a 14 y de 16 a 19 horas